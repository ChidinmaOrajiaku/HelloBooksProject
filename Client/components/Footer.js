import React from 'react';

export default () => {
  return (
    <footer className="page-footer teal">
      <div className="">
        <div className="container">
            Â© 2014 Copyright Text
        </div>
      </div>
    </footer>
  );
};
