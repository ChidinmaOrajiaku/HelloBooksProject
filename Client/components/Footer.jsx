import React from 'react';

export default () => (
  <footer className="page-footer teal">
    <div className="">
      <div className="container">
            Â© 2017 Copyright Text
      </div>
    </div>
  </footer>
);
