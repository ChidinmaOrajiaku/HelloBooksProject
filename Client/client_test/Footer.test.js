import React from 'react';
import $ from 'jquery';
import toJson from 'enzyme-to-json';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import  Footer  from '../components/Footer.jsx';

configure({ adapter: new Adapter() });

describe('<Footer />', () => {
  it('should render component without breaking', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toBeDefined();
  });
  it('should match snapshot test', () => {
    const component = shallow(<Footer />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
