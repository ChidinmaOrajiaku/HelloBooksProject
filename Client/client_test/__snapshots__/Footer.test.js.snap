// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match snapshot test 1`] = `
<footer
  className="page-footer teal"
>
  <div
    className=""
  >
    <div
      className="container"
    >
      Â© 2017 Copyright Text
    </div>
  </div>
</footer>
`;
